src/wwwroot/styles/views/_settings.scss

```diff
@@ -656,19 +656,29 @@
     ol{
       padding-left: 15px;
       margin-bottom: 0px;
```

Comment:
Wasn't this in the reset?

Reply:
checking the reset _normalize.scss I found:
```css
  dl,
  menu,
  ol,
  ul {
    *margin: 1em 0;
  }
```

I need to put zero to the 1em.